dashboard-visualiser-superset/package-metadata.json

```diff
@@ -6,6 +6,8 @@
   "version": "0.0.1",
   "dependencies": ["analytics-datastore-clickhouse"],
   "environmentVariables": {
+    "SUPERSET_IMAGE": "apache/superset:latest",
```

Comment:
Can we please pin this to the latest version by default. We will need to upgrade it periodically.

Reply:
Not 100% sure what you are asking to be honest. Unless someone intentionally overrides this environment variable, it is in essence pinned to the latest version by default no? Or do you want me to explicitly set the image to latest in the compose-file?

Reply:
Sounds good